typespec/Applications.Core/main.tsp

```diff
@@ -41,7 +39,5 @@ interface Operations extends Azure.ResourceManager.Operations {}
 @doc("Supported API versions for the Applications.Core resource provider.")
 enum Versions {
   @doc("2023-10-01-preview")
-  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
-  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
```

Comment:
We don't need them anymore or do they get automatically injected in some other code path?